debian/nnstreamer-protobuf.install

```diff
@@ -1,2 +1,3 @@
 /usr/lib/nnstreamer/converters/libnnstreamer_converter_protobuf.so
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_protobuf.so
+/usr/lib/libnnstreamer_protobuf.so
```

Comment:
/usr/lib/libnnstreamer_protobuf.so
==>
/usr/lib/*/libnnstreamer_protobuf.so

Reply:
Thank you for your guide. Updated.